ligo/haskell/test/Test/Ligo/TreasuryDAO.hs

```diff
@@ -0,0 +1,222 @@
+-- SPDX-FileCopyrightText: 2020 TQ Tezos
+-- SPDX-License-Identifier: LicenseRef-MIT-TQ
+--
+{-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
+-- For all the incomplete list pattern matches in the calls to with
+-- withOriginated func
```

Comment:
Nitpick.
```suggestion
-- For all the incomplete list pattern matches in the calls to the
-- 'withOriginated' function
```

Reply:
This is actually not needed, so I removed this.

Reply:
Ah, even better :ok_hand: